test/module/irohad/consensus/yac/CMakeLists.txt

```diff
@@ -72,21 +62,20 @@ target_link_libraries(yac_timer_test
 addtest(yac_network_test network_test.cpp)
 target_link_libraries(yac_network_test
     yac
-    model
+    shared_model_interfaces
     shared_model_stateless_validation
```

Comment:
Why do you need stateless_validation here?

Reply:
fixed